Simplenote/AuthViewController+Swift.swift

```diff
@@ -367,9 +359,9 @@ extension AuthViewController {
 
         switch superView {
         case usernameField:
-            state.username = usernameField.stringValue
+            state.username = usernameField.stringValue.trimmingCharacters(in: .whitespacesAndNewlines)
         case passwordField:
-            state.password = passwordField.stringValue
+            state.password = passwordField.stringValue.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
Legacy `usernameText` and `passwordText` had this trimming (L97 + L102 in this PR)